rlclientlib/live_model_impl.cc

```diff
@@ -491,33 +502,23 @@ namespace reinforcement_learning {
     return error_code::success;
   }
 
-  int post_process_rank(ranking_response& response, learning_mode learning_mode) {
-    switch (learning_mode) {
-    case IMITATION:
-      {
+  int reset_action_order(ranking_response& response) {
```

Comment:
do we need to do it as std::sort? isn't it always going to be [0, 1, 2, 3, ..] which can be generated with linear complexity given only number of actions?